engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/NodeDataValidator.scala

```diff
@@ -35,7 +35,7 @@ object NodeDataValidator {
         case spel: SpelExpressionParser => spel.typingDictLabels
       }
       val compiler = new NodeCompiler(modelData.processWithObjectsDefinition,
```

Comment:
Maybe better way is create companion object `NodeCompiler` with default `PreventInvocationCollector`?